app/javascript/lib/mixpanel_event_tracking.js

```diff
@@ -0,0 +1,52 @@
+import "../vendor/navigator.sendbeacon.min.js";
+
+
+const MixpanelEventTracking = (function () {
+    const addClickTrackingToOutboundLinks = function () {
+      const links = document.querySelectorAll("a[href]:not([href^='/']):not([href^='#']):not([href^='" + location.protocol + "//" + location.host + "'])");
```

Comment:
✨  praise be to the power of [attribute selectors](https://developer.mozilla.org/en-US/docs/Web/CSS/Attribute_selectors) and [negative selection](https://css-tricks.com/almanac/selectors/n/not/) ✨ !